src/features/defi/components/DefiActionButtons.tsx

```diff
@@ -8,7 +8,7 @@ import { useTranslate } from 'react-polyglot'
 import { matchPath } from 'react-router-dom'
 import { useBrowserRouter } from 'context/BrowserRouterProvider/BrowserRouterProvider'
 
-export const DefiActionButtons = () => {
+export const DefiActionButtons = ({ expired }: { expired: boolean }) => {
```

Comment:
Perhaps we could us a more descriptive name, such as `vaultExpired`, as in the context of the `DefiActionButtons` component it's not clear what has expired.

Reply:
done